pyinteraph/pyinteraph.py

```diff
@@ -168,10 +168,10 @@ def main():
                         default = sbperco_default, \
                         help = sbperco_helpstr.format(sbperco_default))
 
```

Comment:
we should think about the general strategy on how to name files - what if the user uses a .csv filename? do they get a .csv.csv file? Maybe we should just extract the base name of the file (i.e. name without extension) whether the extension is provided or not